README.md

```diff
@@ -17,10 +17,10 @@ A webserver [platform](https://github.com/roc-lang/roc/wiki/Roc-concepts-explain
 
 ## Example
 
-Run this example server with `$ roc helloweb.roc` (on linux, add `--linker=legacy`) and go to `http://localhost:8000` in your browser. You can change the port (8000) and the host (localhost) by setting the environment variables ROC_BASIC_WEBSERVER_PORT and ROC_BASIC_WEBSERVER_HOST.
+Run this example server with `$ roc hello-web.roc` (on linux, add `--linker=legacy`) and go to `http://localhost:8000` in your browser. You can change the port (8000) and the host (localhost) by setting the environment variables ROC_BASIC_WEBSERVER_PORT and ROC_BASIC_WEBSERVER_HOST.
 
 ```roc
-app [Model, server] { pf: platform "https://github.com/roc-lang/basic-webserver/releases/download/0.10.0/BgDDIykwcg51W8HA58FE_BjdzgXVk--ucv6pVb_Adik.tar.br" }
+app [Model, init!, respond!] { pf: platform "../platform/main.roc" }
```

Comment:
I put `platform "<latest release URL from https://github.com/roc-lang/basic-webserver/releases/latest>"` in Weaver to make this less likely to be out of date

Reply:
good catch

Reply:
Fixed